### PR TITLE
Show port description and dns name in FDB table

### PR DESCRIPTION
**Pull request by Jellyfrog**
- Added Port Description field to FDB Table
- Added DNS Name field to FDB Table
- Fixed sorting by port in FDB Table

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
How about adding the field with `data-visible="false"` so it can be displayed if wanted, but is hidden by default.

Also, it feels like this field should be next to the port (interface) field.  That would need to be fixed in the search table too.

**Comment by Jellyfrog** [comment]
> How about adding the field with `data-visible="false"` so it can be displayed if wanted, but is hidden by default. It already displays port (which happens to have ifAlias in the popup).

I think it should be visible by default, its good information.

> Also, it feels like this field should be next to the port (interface) field. That would need to be fixed in the search table too.

Agree, but now it follows how other pages sort the fields, should I add a commit moving the description field for all pages? And maybe rename it to "Port Description" ?

**Comment by jozefrebjak** [comment]
For me working perfectly. Nice feature

**Comment by murrant** [comment]
The reason I suggested hiding it by default is it is duplicate information.

File: html/includes/table/fdb-search.inc.php [hunk 1]
```diff
@@ -56,23 +57,24 @@
...
         $where  .= ' AND `F`.`mac_address` LIKE ?';
         $param[] = $mac_search;
     } else {
-        $where  .= ' AND (`V`.`vlan_vlan` = ? OR `F`.`mac_address` LIKE ?)';
+        $sql .= " LEFT JOIN `ipv4_mac` AS `M` USING (`mac_address`)";
+        $where  .= ' AND (`V`.`vlan_vlan` = ? OR `F`.`mac_address` LIKE ? OR `P`.`ifAlias` LIKE ? OR `M`.`ipv4_address` LIKE ?)';
         $param[] = (int)$search;
         $param[] = $mac_search;
+        $param[] = '%' . $search . '%';
+        $param[] = '%' . mres(gethostbyname(trim($vars['searchPhrase']))) . '%';
```
Comment (laf) [inline html/includes/table/fdb-search.inc.php]: You shouldn't need to use `mres()` here, `$vars` already has done that.

File: html/pages/device/ports/fdb.inc.php [hunk 2]
```diff
@@ -7,7 +7,9 @@
             <th data-column-id="mac_address">MAC address</th>
             <th data-column-id="ipv4_address">IPv4 Address</th>
             <th data-column-id="interface">Port</th>
+            <th data-column-id="description">Description</th>
             <th data-column-id="vlan">Vlan</th>
+            <th data-column-id="dnsname">DNS Name</th>
```
Comment (laf) [inline html/pages/device/ports/fdb.inc.php]: For me this is fairly useless so just has N/A throughout. I imagine it's useful for others but not all.

Comment (Jellyfrog) [reply html/pages/device/ports/fdb.inc.php]: Hidden by default now.

**Pull request merged**